src/internal/interfaces/Visualization.ts

```diff
@@ -168,6 +168,7 @@ export interface IBucketItem {
     overTimeComparisonType?: OverTimeComparisonType;
     operandLocalIdentifiers?: Array<string | null> | null;
     operator?: string | null;
+    dfUri?: string;
```

Comment:
keep attribute, locationDisplayFormUri and dfUri close to each other to represent that these keys are connected with attribute bucket items

Reply:
updated